### PR TITLE
Ask things from user via prompt

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

File: lib/kontena/plugin/vagrant/master/create_command.rb [hunk 1]
```diff
@@ -23,6 +25,18 @@ def execute
       )
     end
 
+    def ask_instance_memory
```
Comment (kke) [inline lib/kontena/plugin/vagrant/master/create_command.rb]: you can use:

```ruby
  option '--foo', 'FOO', 'Foo', default: 'prompt for value'

  def default_foo
    prompt.ask("foo plz?")
  end
end
```

Comment (jakolehm) [inline lib/kontena/plugin/vagrant/master/create_command.rb]: ok, this is clamp magic?

Comment (jakolehm) [inline lib/kontena/plugin/vagrant/master/create_command.rb]: We need a bit more control when these are called.. I think if we use clamp magic we cannot control that? Or are these triggered only when `self.foo` is called?

Comment (jakolehm) [inline lib/kontena/plugin/vagrant/master/create_command.rb]: Actually it's fine in this plugin, more control is needed with other plugins.. will cleanup this a little bit.

**Pull request merged**